packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/template.ts

```diff
@@ -0,0 +1,15 @@
+import { html, when } from '@microsoft/fast-element';
+import type { RichTextMentionUsersView } from '.';
+import { iconAtTag } from '../../../../icons/at';
+
+export const template = html<RichTextMentionUsersView>`<template>
```

Comment:
I'd say simplify the template to just
```
<span class="control"><${iconAtTag}></${iconAtTag}>${x => x.mentionLabel}</span><slot contenteditable="true"></slot>
```

Notes:
- Don't expose the `part` unnecessarily, FAST does it a lot in their templates but fundamentally Nimble is opionated. We don't want to expose unnecessary flexibility which would cause applications to diverge in style.
- At most one instance of the view will have editing enabled, every other instance will have editing disabled. So the template logic overhead is generally extra work / template complexity. Just hide `.control` when editing is disabled with CSS.
- Remove the outer `template` element. It's only needed if you want to bind something to the host. It's not being used for anything so remove it.

Reply:
Updated the template